src/deluge/gui/ui/keyboard/keyboard_screen.cpp

```diff
@@ -535,8 +535,8 @@ ActionResult KeyboardScreen::buttonAction(deluge::hid::Button b, bool on, bool i
 		}
 	}
 
-	else if (b == MIDI) {
-		if (on && currentUIMode == UI_MODE_NONE) {
+	else if (b == MIDI && currentUIMode == UI_MODE_NONE) {
```

Comment:
Should probably preemptively do this for cv too

Reply:
Done!